pvlib/tools.py

```diff
@@ -7,6 +7,7 @@
 import pandas as pd
 import pytz
 import warnings
+from .conftest import DATA_DIR
```

Comment:
This line should be `test_tools`, that will fix the doc build error.

Reply:
Thanks @cwhanse !